client/src/components/EditInfoModal.jsx

```diff
@@ -0,0 +1,276 @@
+import React, {useState} from 'react';
+import ReactModal from 'react-modal';
+import axios from 'axios';
+
+ReactModal.setAppElement('#root');
+const customStyles = {
+  content: {
+    top: '50%',
+    left: '50%',
+    right: 'auto',
+    bottom: 'auto',
+    marginRight: '-50%',
+    transform: 'translate(-50%, -50%)',
+    width: '50%',
+    border: '1px solid #28547a',
+    borderRadius: '4px'
+  }
+};
+
+function EditInfoModal({isOpen, user, handleClose}) {
+  const [showEditModal, setShowEditModal] = useState(isOpen);
+  const [updatedUser, setUpdatedUser] = useState({...user});
+  const [errorMessages, setErrorMessages] = useState('');
+  const [isError, setIsError] = useState(false);
+  const [companySelected, setCompanySelected] = useState('');
+  const [skillSelected, setSkillSelected] = useState('');
+
+  const handleCloseEditModal = () => {
+    setShowEditModal(false);
+    handleClose();
+  };
+
+  const handleChange = (e) => {
+    setUpdatedUser({...updatedUser, [e.target.name]: e.target.value});
+  }
+
+  const handleCompanyChange = (e) => {
+    setCompanySelected(e.target.value);
+  }
+
+  const handleSkillChange = (e) => {
+    setSkillSelected(e.target.value);
+  }
+
+  const handleErrors = (e) => {
+    setErrorMessages(e);
+  }
+
+  const handleSubmit = async (e) => {
+    try{
+      const dataToSend = {
+        ...updatedUser,
+        companyList: companySelected,
+        skillsList: skillSelected
+      }
+      await axios.post(`http://localhost:3000/user/${user.userName}`, dataToSend)
```

Comment:
need to set up start script where we start both fe & be or decide to use hosted versions